pkg/controller/messagingendpoint/messagingendpoint_controller.go

```diff
@@ -344,6 +344,7 @@ func (r *ReconcileMessagingEndpoint) reconcileFinalizer(ctx context.Context, log
 				err = client.DeleteEndpoint(endpoint)
 				if err != nil {
 					if errors.Is(err, stateerrors.ResourceInUseError) {
+						logger.Info("[Finalizer] Endpoint is still in use, rescheduling")
```

Comment:
maybe it should be warn? I'm not sure

Reply:
There is no warn, only info

Reply:
aha ok :D so then merge it! merge it! :D